diagnostic_aggregator/CMakeLists.txt

```diff
@@ -3,9 +3,9 @@ cmake_minimum_required(VERSION 2.8.3)
 project(diagnostic_aggregator)
 
 # Load catkin and all dependencies required for this package
-find_package(catkin REQUIRED diagnostic_msgs pluginlib roscpp rospy rostest xmlrpcpp)
-
-catkin_package(DEPENDS diagnostic_msgs pluginlib roscpp rospy xmlrpcpp
+find_package(catkin REQUIRED diagnostic_msgs pluginlib roscpp rospy rostest xmlrpcpp bondcpp bondpy)
+set(CMAKE_CXX_FLAGS "-g ${CMAKE_CXX_FLAGS}")
```

Comment:
Just noticing this, but I don't think you want to unconditionally add debug symbols here. This would override `CMAKE_BUILD_TYPE` derived flags.